lib/qless/server/static/css/custora_job.css

```diff
@@ -0,0 +1,46 @@
+@media only screen and (max-width: 767px) {
+  .custora-job-label {
```

Comment:
I recommend abbreviating "custora" with something, maybe "c"? Having full names in prefixes tends to bloat code.